lib/apimocker.js

```diff
@@ -123,37 +124,35 @@ apiMocker.setConfigFile = (file) => {
 };
 
 apiMocker.loadConfigFile = () => {
-  if (apiMocker.configFilePath) {
-    apiMocker.log(`Loading config file: ${apiMocker.configFilePath}`);
```

Comment:
Based on this comment, using `require` to load config files has been considered in the past. Having to clear `require`'s cache is certainly an annoyance, but I think the benefits of supporting JS config files justify its usage. `require`'s module caching may even decrease the response time.